requirements/edx-sandbox/base.txt

```diff
@@ -15,7 +15,6 @@ cffi==1.13.2
 git+https://github.com/edx/openedx-chem.git@ff4e3a03d3c7610e47a9af08eb648d8aabe2eb18#egg=chem==1.0.0
 cryptography==2.8
 cycler==0.10.0            # via matplotlib
-enum34==1.1.6
```

Comment:
Note that I remove this also from the sandbox @jmbowman

Reply:
I think I can create a followup ticket to remove this file completely now; it was for Python 2.7 codejails.